GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
@AArnott genuine question... how is the git commit appearing in two places in the nuspec a benefit.  Personally, I don't use the GitVersionTask, but rather I utilize the exe directly.  However, in this instance, I am sure that we could get StyleCop to ignore this file, if we go down that route.

Yes, the yml file is question is the external configuration file for GitVersion.  Is it normally not required, however, if you want to apply some settings to GitVersion, this is where the work is done.